src/lib.rs

```diff
@@ -14,7 +14,7 @@
 
 //! Tremor runtime
 
-#![deny(warnings)]
+// #![deny(warnings)]
```

Comment:
can you comment this in? If this shall be changed, i'd recommend doing this in another PR. (It is totally fine for the Draft, though)